python/remote_api/src/lynx/kite.py

```diff
@@ -854,6 +874,55 @@ def set_executors(self, count):
     return self._send('/remote/setExecutors', {'count': count})
 
 
+def get_free_ports(n):
```

Comment:
Please add a docstring. And make this private.

Reply:
> Please add a docstring. And make this private.

Done.